packages/bpk-component-link/readme.md

```diff
@@ -46,7 +46,10 @@ export default () => (
 
 For both BpkLink and BpkLinkButton the following theme attributes are required.
 
-* `linkTextColor`
+* `linkColor`
 * `linkHoverColor`
 * `linkActiveColor`
+* `linkColor`
```

Comment:
this does not look right, wrong rebase 😄

Reply:
Yeah `linkTextColor` is not correct that seems to have been an error on my part, but you fixed that it seems. Just remove the dupes and we are good